src/main/java/io/debezium/connector/vitess/VitessConnectorConfig.java

```diff
@@ -133,6 +133,14 @@
             .withDescription("Specify a comma-separated list of gRPC headers." +
                     " Defaults to empty");
 
+    public static final Field GRPC_MAX_INBOUND_MESSAGE_SIZE = Field.create(VITESS_CONFIG_GROUP_PREFIX + "grpc.max_inbound_message_size")
+            .withDisplayName("VStream gRPC maxInboundMessageSize")
+            .withType(Type.INT)
+            .withWidth(Width.SHORT)
+            .withDefault(0)
```

Comment:
Default is 4194304 (4 MiB).
https://grpc.github.io/grpc-java/javadoc/io/grpc/ManagedChannelBuilder.html#maxInboundMessageSize-int-